codeandtheory/terraform-modules
# DEVOPS-91 - upgrade terraform

# Related Tickets

https://codeandtheory.atlassian.net/browse/DEVOPS-91

# Description

There are 3 things happening in this PR
1. I updated the S3 policy being generated by the `s3` module to only allow listing content in the bucket being created
2. I ran `terraform fmt` to format all the modules better
3. I removed a `template` directory that needs an update

# Validation


1. Create a bucket and policies using the `s3` module.
2. Create an IAM test user and grant that user the policies generated by the `s3` module
3. Verify that only the bucket created is accessible

This is the only thing that needs to be validated


# Risks

None - although existing policies created by the `s3` module, if they exist, needs to be updated.
